core/invoker/src/main/scala/whisk/core/container/WhiskContainer.scala

```diff
@@ -92,17 +96,53 @@ class WhiskContainer(
     }
 
     /**
+     * Starts the tracing.
+     */
+    private def startSpan(msg: ActivationMessage): Option[Span] = {
+        val tracerOption = if (GlobalTracer.isRegistered) Option.empty else Some(TracerResolver.resolveTracer)
```

Comment:
actually, this doesn't work because 
```scala
Option(GlobalTracer.isRegistered).exists(identity)
```
returns either `true` or `false` and I can't call map on Boolean

this works:

```scala
Option(GlobalTracer.isRegistered).flatMap(if (_) Option.empty else Option(TracerResolver.resolveTracer))
```

but perhaps it's too much "flatmappy" :)